src/s_sound.c

```diff
@@ -242,6 +264,10 @@ static int S_getChannel(const mobj_t *origin, sfxinfo_t *sfxinfo,
    int cnum;
    int lowestpriority = -1; // haleyjd
    int lpcnum = -1;
+   int instances = 0;
+
+   // Sort the sound channels by descending priority levels
+   SDL_qsort(channels, numChannels, sizeof(channel_t), S_CompareChannels);
```

Comment:
`SDL_qsort` crashes in MSVC build (works in MSYS). A simple `qsort` works for me. Should we use `SDL_qsort`? It's not documented in https://wiki.libsdl.org/